README.md

```diff
@@ -7,4 +7,104 @@ $ make build
 $ make run-local
 ````
 
-_"We dream of a world in which no more squids will be needlessly slaughtered for their ink. - Arthur Troy Astorino III"_
+# Mission:
+```
+Package the SplitMe!TM application into a Helm chart, 
+and use Terraform to deploy that Helm chart into a 
+Kubernetes cluster running in minikube or a cloud environment.
+```
+
+## Observations:
```

Comment:
Could you please add some observations about how the application functions and how it can be improved to serve millions of users nationwide?